Fix two qa warnings due to "implicit declaration of function _exit()" which
we can find into "unistd.h", and "implicit declaration of function exit()" which
we can find into "stdlib.h"

 gnome-swallow.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/gnome-swallow.c b/gnome-swallow.c
index 143ef1c..cb4d38a 100644
--- a/gnome-swallow.c
+++ b/gnome-swallow.c
@@ -1,5 +1,7 @@
 #include <string.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
 #include <signal.h>
 #include <sys/wait.h>
 
